### DISCUSSION TITLE
Does uno support EF Core, MVVM Toolkit?

### DISCUSSION THREAD

**Discussion by syaifulnizamyahya** [unoplatform/uno#9165] [Q&A] [answered]
I plan to build an application that uses UWP(instead of MAUI). Reason being MAUI is very new and has lots of missing features that I required. My target platform is 
1. Android
2. ios
3. Web
4. Windows

I would like to know if Uno support 
1. EF Core, https://docs.microsoft.com/en-us/ef/core/miscellaneous/platforms#universal-windows-platform 
2. MVVM Toolkit, https://docs.microsoft.com/en-us/windows/communitytoolkit/mvvm/introduction

Can I use any .net library supported by UWP in Uno? What are the restriction for the compatibility(eg. if the library requires loading assemblies, ios does not support it etc)?

**Comment by jeromelaban** [comment, marked as answer]
Uno Platform relies on what .NET provides underneath (Xamarin, WebAssembly or .NET 6). In general, the support for each library depends their existing target frameworks availability. Any `netstandard2.0` library is generally supported, which is the case for MVVM Toolkit.

For EF Core, it depends on the features you'll want to use and the target platform you want to use. I suggest you give it a try to find out what works for you.

**Reply by syaifulnizamyahya** [reply]
Ok. Got it. Any netstandard2.0 library are most likely will be supported on all platforms.
